src/app/solicitudes/list-solicitudes-estudiante/list-solicitudes-estudiante.component.ts

```diff
@@ -93,7 +93,7 @@ export class ListSolicitudesEstudianteComponent implements OnInit {
         .get('solicitudes-evaluacion/estados/' + IdEstadoTipoSolicitud)
         .subscribe(
           (response: any) => {
-            if (response.status === 200) {
+            if (response.Status === 200) {
               const data = <Array<any>>response.data.Resultado;
```

Comment:
Me parece que falta el data como Data
```suggestion
              const data = <Array<any>>response.Data.Resultado;
```